appengine/flexible/Datastore/runTests.ps1

```diff
@@ -12,12 +12,9 @@
 # License for the specific language governing permissions and limitations under
 # the License.
 
-Import-Module -DisableNameChecking ..\..\..\BuildTools.psm1
-
 dotnet restore --force
 BackupAndEdit-TextFile "appsettings.json" `
```

Comment:
Instead of doing this, and adding the the value to the appsettings.json, you can just use the env variable directly in code if you do `builder.Configuration["GOOGLE_PROJECT_ID"]`.

And maybe you can get rid of appsettings.json entirely.

Reply:
Tests are not beeing run now but I think that's because `appsetings.json` is one level below runTests.ps1 and it's not being found. If you make the changes for using the env var directly, and remove the step to edit `appsettings.json` they should run.

Reply:
Done for both.